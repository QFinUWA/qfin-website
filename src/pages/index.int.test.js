import React from 'react';
import { render } from '@testing-library/react';
import Home from './index';

describe('<Home/>', () => {
  it('matches the snapshot', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  })
})