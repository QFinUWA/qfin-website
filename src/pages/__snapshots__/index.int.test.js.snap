// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home/> matches the snapshot 1`] = `
<div>
  <div
    class="page"
  >
    <h1
      class="title"
    >
      Welcome to QFin
    </h1>
  </div>
</div>
`;
